sdk/keyvault/keyvault-admin/recordings/node/keyvaultaccesscontrolclient/recording_listroleassignments.js

```diff
@@ -0,0 +1,59 @@
+let nock = require('nock');
+
+module.exports.hash = "40a7c9b72f55641675802c7541f1b95b";
+
+module.exports.testInfo = {"uniqueName":{},"newDate":{}}
+
+nock('https://eastus2.keyvault_name.managedhsm.azure.net:443', {"encodedQueryParams":true})
+  .get('///providers/Microsoft.Authorization/roleAssignments')
+  .query(true)
+  .reply(401, "OK", [ 'content-type',
+  'application/json; charset=utf-8',
+  'x-content-type-options',
+  'nosniff',
+  'www-authenticate',
+  'Bearer authorization="https://login.windows-ppe.net/azure_tenant_id", resource="https://managedhsm-int.azure-int.net"',
```

Comment:
I was unable to obscure this instance. .Net doesn't obscure this instance at all, so I'm not planning on going more in depth.